singolo/sass/style.scss

```diff
@@ -1,3 +1,7 @@
+*{
```

Comment:
What if we create a common.scss file for common style and classes?

Reply:
Yes, it's better that way

Reply:
It's better this way